src/app/components/project-map/project-map.component.html

```diff
@@ -11,6 +11,7 @@
     [selection-tool]="tools.selection"
     [moving-tool]="tools.moving"
     [text-editing-tool]="tools.text_editing"
+    [text-adding-tool]="drawTools.isAddingTextChosen"
```

Comment:
What do you think if we use service for tools activation/deactivation? It would be based on observable.